app.py

```diff
@@ -4,7 +4,7 @@
 
 def create_app() -> Flask:
     flask_app = Flask(__name__)
-    flask_app.config.from_pyfile("config.py")
+    flask_app.config.from_object("config.default.Default")
```

Comment:
Does this need the FLASK_ENV environment switching here?

Reply:
This is moved into __init__.py now